src/apipkg/__init__.py

```diff
@@ -9,7 +9,8 @@
 import sys
 from types import ModuleType
 
-from .version import version as __version__
+#from .version import version as __version__
+__version__ = '1.5'
```

Comment:
that has to go ^^

Reply:
Ha, yes indeed. That does not belong in the PR. Will clean this up in the morning and experiment in a fresh branch.

Reply:
Ha, yes indeed. That does not belong in the PR. Will clean this up in the morning and experiment in a fresh branch.